permissions/src/iosMain/kotlin/dev/icerock/moko/permissions/ios/MotionPermissionDelegate.kt

```diff
@@ -0,0 +1,30 @@
+package dev.icerock.moko.permissions.ios
```

Comment:
add copyright like in other files please

Reply:
Added copyright.